src/holodeck/weather.py

```diff
@@ -10,6 +10,7 @@ class WeatherController:
     """
     def __init__(self, send_world_command):
         self._send_command = send_world_command
+        self.cur_weather = "sunny"
```

Comment:
This was added so the user can check the current weather in the environment if necessary. The default value should probably not be hard coded as sunny, but for the moment we don't really have a way to find the default weather from the world. I think its fine since it would be pretty weird if a world was not sunny by default.